accession-commons-monotonic-generator-jpa/src/test/resources/application-test1.properties

```diff
@@ -1 +1,3 @@
 accessioning.monotonic.init.category-id-monotonic-test = 100
+accessioning.monotonic.init.eva = 1
+accessioning.monotonic.init.dbSNP = 6
```

Comment:
The problem is that both EVA and dbSNP will issue accessions with the same prefix... Maybe you could define a property for how long the gap between 2 EVA blocks have to be instead.

Reply:
Sorry , ya we can have the prefix to be same , the block gap is specified in nextBlockInterval parameter in reserveNewBlock . so now the eva accessions will be eva1 .. eva5,eva11..eva15 and dbsnp accessions would be dbsnp6..dbsnp10,dbsnp16..dbsnp20 we can change the prefix of dbsnp to be eva. is that what you expect ?